tests/Doctrine/Tests/Common/Annotations/Fixtures/ClassWithClosure.php

```diff
@@ -26,7 +26,7 @@
     public function methodName(\Closure $callback)
     {
         $self = $this;
-        return function() use ($self, $callback) {
+        return function() use ($callback) {
```

Comment:
It's in a separate commit with description - `$self` is unused inside the closure. I could probably also remove the `$self = $this` line. PHPStan also detects unused constructor parameters in a similar fashion - what can be removed without consequences, or was probably forgotten to be used (like injected parameter in the constructor that's not injected into a property).